drivers/lcd/memlcd.c

```diff
@@ -570,6 +574,9 @@ static int memlcd_getrun(FAR struct lcd_dev_s *dev,
           *buffer &= ~usrmask;
         }
 
+#if CONFIG_MEMLCD_BYTE_PER_PIXEL
+      buffer++;
+#else
 #ifdef CONFIG_LCD_PACKEDMSFIRST
```

Comment:
can go with
```suggestion
#elif defined(CONFIG_LCD_PACKEDMSFIRST)
```
end eliminate one level of ifdef

Reply:
You are right. Thanks.